src/main.rs

```diff
@@ -1,18 +1,21 @@
-//! Darwinia Node CLI
+//! Darwinia CLI
 
 #![warn(missing_docs)]
 
-fn main() -> sc_cli::Result<()> {
-	let version = sc_cli::VersionInfo {
-		name: "Darwinia Node",
+// --- darwinia ---
+use darwinia_cli::VersionInfo;
+
+fn main() -> Result<(), darwinia_cli::Error> {
+	let version = VersionInfo {
+		name: "Darwinia-Network Darwinia",
 		commit: env!("VERGEN_SHA_SHORT"),
 		version: env!("CARGO_PKG_VERSION"),
 		executable_name: "darwinia",
 		author: "Darwinia Network <hello@darwinia.network>",
-		description: "Darwinia node implementation in Rust",
+		description: "Darwinia Relay-chain Client Node",
```

Comment:
We do not act as Relay Chain any more, change to "Darwinia node implementation in Rust"